cassandra/cqltypes.py

```diff
@@ -904,10 +907,16 @@ def serialize_safe(cls, themap, protocol_version):
         for key, val in items:
             keybytes = key_type.to_binary(key, inner_proto)
             valbytes = value_type.to_binary(val, inner_proto)
-            buf.write(pack(len(keybytes)))
-            buf.write(keybytes)
-            buf.write(pack(len(valbytes)))
-            buf.write(valbytes)
+            if keybytes is not None:
```

Comment:
Nitpick: shouldn't you check if key is None, instead of first converting it to binary and checking if the result is None?

Reply:
It's correct either way, as `to_binary` checks for `None`, but your way may be slightly faster as it avoids unnecessary function call. I'll change that